src/uct/ib/base/ib_iface.h

```diff
@@ -193,6 +193,7 @@ typedef struct uct_ib_iface_init_attr {
     unsigned    fc_req_size;            /* Flow control request size */
     int         qp_type;                /* IB QP type */
     int         flags;                  /* Various flags (see enum) */
+    int         disable_cqe_zipping;    /* Allows to enable CQE zipping */
```

Comment:
1. enable_cqe_zipping
2. it should be mlx5 config only since it's mlx5 specific offload, not generic verbs.

Reply:
1. Pls see my comment [here](https://github.com/openucx/ucx/pull/7992#discussion_r828800481).
2. Removed